docs/content/en/docs/references/api/grpc.md

```diff
@@ -670,6 +675,74 @@ Suggestion defines the action a user needs to recover from an error.
 
 
 
+<a name="proto.TestEvent"></a>
+#### TestEvent
+`TestEvent` represents the status of a test, and is emitted by Skaffold
+anytime a test starts or completes, successfully or not.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| status | [string](#string) |  | test status oneof: InProgress, Completed, Failed |
+| err | [string](#string) |  | Deprecated. Use actionableErr.message. error when status is Failed |
```

Comment:
i think we can probably just not add this since it's deprecated? we've already got the `actionableErr` here so we should be able to just use that. cc @tejal29

Reply:
With the new Event API V2, all of this is going to go away. 
See https://docs.google.com/document/d/1SgxQNJsMD8u1P4rh2Ln-Lw7tz3PlBDUJkRSA5A61q7U/edit#

I am not sure, if it makes to add these and IDEs not consuming these at all.

Reply:
Removed the deprecated fields.